rest_api/src/main/resources/static/assets/js/login.js

```diff
@@ -0,0 +1,40 @@
+var base_api_url = "http://localhost:8080";
```

Comment:
```suggestion
var base_api_url = "http://192.168.160.63:8080";
```
Same applies for other js files